hiddenregions.xml

```diff
@@ -0,0 +1,79 @@
+<!--
```

Comment:
make this a lua comment instead of xml so that it ports to sm5 more smoothly

Reply:
you did this just now